### PR TITLE
Automatic Translations using Google Translate

### PR DESCRIPTION
**Pull request by jfangrad**
This adds in a script to run automatic translations on any new translations that have been added to the `en.json` translation dictionary. The translations are done using Google translate through [this package](https://github.com/k3rn31p4nic/google-translate-api)

To use, just run `yarn translate`.

The script will output summary data for each file in the console including [# of success] | [# of failed] | [failedKeys] | [successfulKeys] |

It will also update the coresponding changlog textfile to keep track of which keys where translated through Google translate so that when they are checked by a human they can see which keys they need to check.

### CODE REVIEW

**Comment by jfangrad** [comment]
[Automated Translations Using Google Translate](https://trello.com/c/MvHJRLBT/167-automated-translations-using-google-translate)

File: scripts/translations/getTranslations.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/*
+  eslint
+  no-console: 0,
+*/
+const translate = require('@k3rn31p4nic/google-translate-api'); // @vitalets/google-translate-api
```
Comment (smaclell) [inline scripts/translations/getTranslations.js]: I am a little weirded out by the scoped package. Why not https://www.npmjs.com/package/google-translate-api directly?

File: scripts/translations/getTranslations.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+const folderPath = path.join(__dirname, '..', '..', 'assets', 'i18n', 'locales');
+const files = fs.readdirSync(folderPath);
+const stats = {};
+
+function getLocale(file) {
+  let locale = file.split('.')[0];
+  if (locale === 'zh-hk') {
+    locale = 'zh-cn'; // The other option we have available is zh-tw (Traditional chinese), idk which is more similar to chinese (Hong-Kong)
+  }
+  if (locale === 'pt-br') {
```
[the remark this answers is not in the document]
Comment (jfangrad) [reply scripts/translations/getTranslations.js]: It's not that the ones we are using aren't known locales, it's more that these specialized ones aren't supported by the translation library.

File: scripts/translations/getTranslations.js (hunk 1 above)
Comment (jfangrad) [reply scripts/translations/getTranslations.js]: That one isn't supported anymore and didn't work for me. There are a couple of supported forked versions and this was one of them and the only one that I could get to work consistently.

**Pull request merged**